### PR TITLE
user view subject

### PR DESCRIPTION
**Pull request by hienmvvn**

### CODE REVIEW

File: app/controllers/courses_controller.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class CoursesController < ApplicationController
+  before_action :find_course
+  def show
+    @user_subjects = @course.user_subjects
+  end
+  
+  private
+  def find_course
+    @course = UserCourse.find(params[:id])
```
Comment (tanphan313) [inline app/controllers/courses_controller.rb]: bỏ ngoặc

File: app/controllers/courses_controller.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+class CoursesController < ApplicationController
+  before_action :find_course
```
Comment (ngocquyhoang) [inline app/controllers/courses_controller.rb]: block code

File: app/controllers/users_controller.rb [hunk 3]
```diff
@@ -1,5 +1,5 @@
 class UsersController < ApplicationController
   def show
-    @courses = current_user.courses
+
```
Comment (tanphan313) [inline app/controllers/users_controller.rb]: xóa dòng

File: app/controllers/static_pages_controller.rb [hunk 4]
```diff
@@ -1,4 +1,5 @@
 class StaticPagesController < ApplicationController
   def home
+  	 @courses = current_user.user_courses
```
Comment (ngocquyhoang) [inline app/controllers/static_pages_controller.rb]: indent

File: app/views/static_pages/home.html.erb [hunk 5]
```diff
@@ -1,9 +1,16 @@
 <div class="center jumbotron">
+<% if user_signed_in? %>
+  <h3><%= t("user.my_course") %> </h3>
```
Comment (ngocquyhoang) [inline app/views/static_pages/home.html.erb]: remove ()

File: app/controllers/static_pages_controller.rb (hunk 4 above)
Comment (tanphan313) [inline app/controllers/static_pages_controller.rb]: tab

File: app/views/courses/show.html.erb [hunk 6]
```diff
@@ -0,0 +1,4 @@
+<h3><%= t("user.my_subject")%></h3>
```
Comment (tanphan313) [inline app/views/courses/show.html.erb]: bỏ ()

File: app/views/static_pages/home.html.erb [hunk 7]
```diff
@@ -1,9 +1,16 @@
...
+  <% @courses.each do |user_course| %>
+    <p class="button">
+      <%= user_course_status user_course %>
+    </p>
+  <% end %>
+<% else %>
   <h1><%= t "home_page.welcom" %></h1>
   <h2>
     <%= t "home_page.welcom1" %>
     <a href="http://www.railstutorial.org/"><%= t "ruby_on_rails" %></a><%= t "home_page.welcom2" %>
```
Comment (ngocquyhoang) [inline app/views/static_pages/home.html.erb]: 80 character

File: app/controllers/courses_controller.rb [hunk 8]
```diff
@@ -0,0 +1,13 @@
...
+
+  def show
+    @user_subjects = @course.user_subjects
+  end
+  
+  private
+  def find_course
+    @course = UserCourse.find params[:id]
+  end
+
```
Comment (ngocquyhoang) [inline app/controllers/courses_controller.rb]: delete line

File: app/controllers/static_pages_controller.rb [hunk 9]
```diff
@@ -1,4 +1,5 @@
 class StaticPagesController < ApplicationController
   def home
+  	@courses = current_user.user_courses
```
Comment (ngocquyhoang) [inline app/controllers/static_pages_controller.rb]: indent

**Comment by ngocquyhoang** [comment]
OK

**Pull request merged**